lib/src/chart/line_chart/line_chart_helper.dart

```diff
@@ -46,20 +46,24 @@ class LineChartHelper {
     var maxY = firstValidSpot.y;
 
     for (var barData in lineBarsData) {
-      if (barData.mostRightSpot.x > maxX) {
-        maxX = barData.mostRightSpot.x;
-      }
-
-      if (barData.mostLeftSpot.x < minX) {
-        minX = barData.mostLeftSpot.x;
-      }
-
-      if (barData.mostTopSpot.y > maxY) {
-        maxY = barData.mostTopSpot.y;
-      }
-
-      if (barData.mostBottomSpot.y < minY) {
-        minY = barData.mostBottomSpot.y;
+      try {
+        if (barData.mostRightSpot.x > maxX) {
```

Comment:
I think it's better to put an `if check` at the beginning instead of `try catch`. Something like:
```dart
if (barData.isEmpty) {
   continue;
}
```

Reply:
I mean it is more readable.